src/platform/windows/publish.cpp

```diff
@@ -29,7 +29,7 @@ using namespace std::literals;
 #define SV(quote) __SV(quote)
 
 extern "C" {
-constexpr auto DNS_REQUEST_PENDING = 9506L;
+constexpr auto MY_DNS_REQUEST_PENDING = 9506L;
```

Comment:
We don't really name variables "my_variable".

Reply:
This was the change I had to make for the CI, it was failing with this error otherwise - https://github.com/utkarshdalal/Sunshine-GameAway/actions/runs/5632865670/job/15261219026. Screenshot below.
<img width="1728" alt="image" src="https://github.com/LizardByte/Sunshine/assets/3746571/95404174-08a8-4e0d-a6ae-55fd5bc7df55">

I can name it something else also if you like, but this variable name was conflicting with a reserved variable name.